ostree-upload-server.py

```diff
@@ -12,22 +15,28 @@
 
 from flask import Flask, jsonify, request, render_template, send_from_directory
 
-PORT = 5000
+MAINTENANCE_WAIT = 10
 
 
 # TODO: Turn this into a class
 def worker(queue, quit):
+    global latest_task_complete
```

Comment:
This can now be moved into a Workers class member, I made a note so this doesn't get lost.